api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessConstants.java

```diff
@@ -14,4 +14,6 @@
 
     public static final  String LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME = "locationbasedaccess.locationAttributeUuid";
     public static final  String LOCATION_USER_PROPERTY_NAME = "locationUuid";
+
+    public static final String REF_APP_LOCATION_USER_PROPERTY_NAME = "referenceapplication.locationUserPropertyName";
```

Comment:
Of what value is this GP when you expect the user property to be named as "locationUuid"?

Reply:
exactly "locationUuid" as the GP value

Reply:
We do not do global properties for fixed values that we know. Otherwise, of what value would the gp be?

Reply:
So can we move the locationUserProperty to a global property name as "locationbasedaccess.locationUserPropertyName"? So the userProperty name also will be fetched from the GP, and use that same value to this RefApp GP as well?

Reply:
What do you think? What are the pros and cons of either side?

Reply:
The pros will be making this more flexible to the users since they can use their own property names. If some implementers used this propertyName(locationUuid) for some other reasons, they can also use different name through this feature

Cons : we are depending on so many GPs, and better to automate this through the module config. 

What do you think?

Reply:
So which would you prefer?

Reply:
As you mentioned, we can't use the GP for the static values. So would be better to go for the GP option to the location userPropertyName as well. Can I create another ticket in LBAC to make this?

Reply:
OK.

Reply:
No need to create new one, Will reopen this : https://issues.openmrs.org/browse/LBAC-2

Reply:
We decided to drop this idea and go with it for now. So are there any more comments?

Reply:
So what are you using the gp for?

Reply:
We need this to connect our module with RefApp module without any direct coupling. Since both need this userPropertyName. So we can connect them through this GP. 
Otherwise we need to hardcode this value in both module.

Reply:
Got you! 😊